rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/mapper/WorkbasketSummaryResourcesAssembler.java

```diff
@@ -38,7 +38,22 @@ public WorkbasketSummaryResourcesAssembler() {
 
         UriComponentsBuilder original = getBuilderForOriginalUri();
         pagedResources.add(new Link(original.toUriString()).withSelfRel());
-        pagedResources.add(linkTo(WorkbasketController.class).withRel("allWorkbaskets"));
+        if (pageMetadata != null) {
```

Comment:
I didn't expect to get previous, next, first and last pages, It's nice having that :)

Reply:
I thought this would be nice for our HATEOAS concept. But you are probably right and I overengineered this part a little bit... :-)